pyproject.toml

```diff
@@ -48,4 +46,20 @@ line_length = 105
 src_paths = ["gtfs"]
 skip = [
     ".venv"
+]
+
+[tool.mypy]
+strict_optional = true
+warn_return_any = true
+warn_redundant_casts = true
+check_untyped_defs = true
+disallow_any_generics = true
+no_implicit_optional = true
+warn_unused_configs = true
+show_error_codes = true
+show_column_numbers = true
+exclude = [
+    '\.git',
+    '__pycache__',
+    'venv/',
```

Comment:
similar here: a short description as comments would be great. makes it more re-usable as well.